git2-hooks/Cargo.toml

```
@@ -14,6 +14,7 @@ keywords = ["git"]
 
 [dependencies]
 git2 = ">=0.17"
+gix-path = "0.10.15"
```

Let's not depend on the exact patch version here, same as for the other dependencies.

```suggestion
gix-path = "0.10"
```